db/migrate/20170414132105_add_language_to_statuses.rb

```diff
@@ -0,0 +1,5 @@
+class AddLanguageToStatuses < ActiveRecord::Migration[5.0]
+  def change
+    add_column :statuses, :language, :string, null: false, default: 'en'
```

Comment:
Will this make all historical statuses on an instance retroactively call themselves `en` ?